src/DateTime/DateTime.py

```diff
@@ -680,6 +680,8 @@ def _parse_args(self, *args, **kw):
             # flag indicating whether this was constructed in a timezone naive
             # manner
             yr, mo, dy, hr, mn, sc, tz, t, d, s, microsecs, tznaive = args
+            if isinstance(microsecs, float):
+                microsecs = long(round(microsecs))
```

Comment:
I see `NameError: name 'long' is not defined` on Python 3. Does this branch only get executed on Python 2? At least a comment would be helpful.

Reply:
On the top of this file there is `long = int` when run on Python 3. `long` is already used 35 times in this file.